GameOfHive/ViewController.swift

```diff
@@ -297,6 +302,10 @@ class ViewController: UIViewController {
             return
         }
         
+        cells.forEach { cell in
+            cell.userInteractionEnabled = true
+        }
```

Comment:
DRY als het niet alleen op contentview kan maar idd per cell dan liever in een functie (met bool argument) en die hier en de andere locatie aanroepen